src/Verilator.cpp

```diff
@@ -690,6 +691,9 @@ static bool verilate(const string& argString) {
     // Initialize thread pool
     v3Global.threadPoolp(new V3ThreadPool{v3Global.opt.verilateJobs()});
 
+    // Initialize library mappings
+    v3Global.libMapp(new V3LibMap());
```

Comment:
Nit.  However is this better done in the v3global constructor?

```suggestion
    v3Global.libMapp(new V3LibMap{});
```